diff --git a/node_modules/@10up/block-components/dist/index.js b/node_modules/@10up/block-components/dist/index.js
index 2b9f2b7..06c01d1 100644
--- a/node_modules/@10up/block-components/dist/index.js
+++ b/node_modules/@10up/block-components/dist/index.js
@@ -3579,6 +3579,7 @@ const MediaToolbar = ({
       columnNumber: 3
     }
   }, hasImage ? (0,_wordpress_element__WEBPACK_IMPORTED_MODULE_0__.createElement)(_wordpress_element__WEBPACK_IMPORTED_MODULE_0__.Fragment, null, (0,_wordpress_element__WEBPACK_IMPORTED_MODULE_0__.createElement)(_wordpress_block_editor__WEBPACK_IMPORTED_MODULE_2__.MediaReplaceFlow, {
+    mediaId: id,
     mediaUrl: media?.source_url,
     onSelect: onSelect,
     name: mergedLabels.replace,
